client/landing/stepper/declarative-flow/copy-site.tsx

```diff
@@ -104,6 +97,7 @@ const copySite: Flow = {
 					/>
 				),
 			},
+			{ slug: 'resuming', component: ProcessingStep }, // Needs siteSlug param
```

Comment:
These steps don't need to be sequential.
I removed the dynamic steps changing and added a new `resuming` step.
So the link on `SiteResumeCopy`  is `/setup/copy-site/resuming?...`

https://github.com/Automattic/wp-calypso/blob/82550199a42b66f2ad2a62e8c5e6818973027508/client/my-sites/customer-home/cards/tasks/site-resume-copy/index.jsx#L33-L36

Reply:
Nice TIL, thanks! It looks much cleaner now.